packages/components/src/components/popover/popover.tsx

```diff
@@ -29,38 +30,56 @@ const Popover = ({
     relative_render = false,
     should_disable_pointer_events = false,
     should_show_cursor,
-    window_border,
     zIndex = '1',
     data_testid,
     arrow_styles,
 }: React.PropsWithChildren<TPopoverProps>) => {
     const ref = React.useRef<HTMLDivElement | undefined>();
     const [popover_ref, setPopoverRef] = React.useState<HTMLDivElement | undefined>(undefined);
-
+    const [is_bubble_visible, setIsBubbleVisible] = React.useState(false);
+    const {
+        ui: { is_mobile },
+    } = useStore();
     const [hover_ref, is_hovered] = useHover(null, true);
     const [bubble_hover_ref, is_bubble_hovered] = useHoverCallback();
+    const should_toggle_on_target_tap = React.useMemo(() => is_mobile && is_open === undefined, [is_mobile, is_open]);
```

Comment:
For responsive only and for uncontrolled popovers only (when is_open is not provided).